static/js/context-menu.js

```diff
@@ -866,3 +899,144 @@ function getTargetWord() {
 
   return string.substring(start, end);
 }
+
+var TREE_LIST = [
```

Comment:
Requesting changes: We should extract the TREE_LIST out into a JS file that can live in the root of the config directory.  This is not only a win for any other mozsearch instances out there, but also will make it easier for people making changes to mozsearch-mozilla to not miss the need to update the tree list.

It might makes sense to defer making this change until after getting some accessibility feedback if it ends up making more sense to let this just be an HTML snippet or something.